queries/t32/indents.scm

```diff
@@ -0,0 +1,36 @@
+; Subroutines & scoping
+(block
+  ")" @indent.branch .) @indent.begin
+
+
+; Control flow statements
+(
+  (if_block
+    _*
+    condition: (_)
+    . (_) @_then) @indent.begin
+  (#not-has-type? @_then block)
+)
+
+(else_block) @indent.branch
+
+(else_block
+  (if_block) @indent.dedent) @indent.branch
+
+(
+  (else_block
+    (if_block
+      _*
+      condition: (_)
+      . (_) @_then)) @indent.branch
+  (#not-has-type? @_then block)
+)
+
+(while_block
+  (command_expression) @indent.end .) @indent.begin
```

Comment:
Fwiw, with the current indent logic, this indent.end capture will most likely never matter.

indent.end must be the final and smallest range possible node of a line, i.e. ";" in C, "}"

Reply:
Note that indent.end as of now is strictly just for interactive typing. It has no effect on the result of "gg=G"

Reply:
This query is only covering the most basic case where we have a single command without delimiters after the IF:
```
WHILE &a
  Step
```

Removing `@indent.end` does not change the behavior, so it is gone now. I will probably have to revisit these queries in the future, if I get a better understanding of the differences between interactive typing and full file indentation.